### PR TITLE
Add integer fields

### PR DESCRIPTION
**Pull request by KevinEtienne**

### CODE REVIEW

**Comment by KevinEtienne** [comment]
@adam-incuna do you want to review please?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,14 @@
+## Upcoming
+
+* Rename the following fields:
+  `PGPPublicKeyField` to `TextPGPPublicKeyField`;
+  `PGPSymmetricKeyField` to `TextPGPSymmetricKeyField`;
+  `FieldHash` to `TextFieldHash`;
```
Comment (adam-incuna) [inline CHANGELOG.md]: Should this be `DigestField`? I'm slightly confused.

Comment (KevinEtienne) [inline CHANGELOG.md]: :+1:

**Comment by adam-incuna** [comment]
This seems reasonable, although I haven't worked with `pgcrypto` so I have no real idea on how it works.  Travis seems happy and coverage is still 100%, so I'm gonna go ahead and assume I haven't missed anything glaring :+1:

**Comment by KevinEtienne** [comment]
Thanks!

**Pull request merged**